app/src/main/java/com/example/lhi06/residue/AlbumReviewActivity.java

```diff
@@ -38,17 +38,17 @@ protected void onCreate(@Nullable final Bundle savedInstanceState) {
 
         final Album album = getIntent().getExtras().getParcelable(FindAlbumActivity.ALBUM_EXTRA);
         if (album != null) {
-            albumName.setText(album.getCollectionName());
-            artistName.setText(album.getArtistName());
-            Picasso.with(this).load(album.getArtworkUrl()).into(albumArtwork);
+            albumNameTextView.setText(album.getCollectionName());
+            artistNameTextView.setText(album.getArtistName());
+            Picasso.with(this).load(album.getArtworkUrl()).into(albumArtworkImageView);
         }
         saveReviewButton.setOnClickListener(new View.OnClickListener() {
```

Comment:
This needs to be inside the album null check block as it uses the `album` variable and expects it to be non-null